hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -4,9 +4,9 @@
  * Collection of upgrade steps
  */
 class CRM_Hrjobcontract_Upgrader extends CRM_Hrjobcontract_Upgrader_Base {
-
+  private $_enities;
```

Comment:
s/enities/entities/

Also, never prefix private properties with `_`. This was something people did when PHP didn't have the access modifiers.